src/entity/Company.ts

```diff
@@ -25,4 +30,10 @@ export class Company {
 
   @Column({ type: 'text', array: true, default: [] })
   domains: string[];
+
+  @Column({
```

Comment:
Overall note you are missing migration file for psql changes

Reply:
I'm missing something from [this](https://github.com/dailydotdev/daily-api/pull/2944/files#diff-e06fe0d4808cc58d47217607e972ee2d664d161fb0605b65fc420ed45d60496c)?